includes/admin/class-amp-customizer.php

```diff
@@ -86,7 +87,14 @@ public function add_customizer_scripts() {
 			$footer  = true
 		);
 
-		wp_localize_script( 'amp-customizer', 'ampPost', amp_admin_get_preview_permalink() );
+		wp_localize_script( 'amp-customizer', 'ampVars', array(
```

Comment:
Instead of adding a global variable, I suggest that the `amp-customizer` script export a boot method which we can then pass an inline script into. For example: 

https://github.com/Automattic/amp-wp/blob/eeb502b28807c1be80e59937fab6112e406b204b/assets/js/amp-post-meta-box.js#L51-L59

https://github.com/Automattic/amp-wp/blob/eeb502b28807c1be80e59937fab6112e406b204b/includes/admin/class-amp-post-meta-box.php#L95-L111

This will avoid an additional global var, it will make the JS more encapsulated and easier to test, and `wp_localize_script()` has the nasty side effect of converting an scalar values to strings (which wouldn't apply here though).